addons/zookeeper/templates/cmpd.yaml

```diff
@@ -43,27 +44,63 @@ spec:
       serviceable: false
       writable: false
       votable: false
+  updateStrategy: BestEffortParallel
+  vars:
+    - name: ZOOKEEPER_POD_FQDN_LIST
+      valueFrom:
+        componentVarRef:
+          optional: false
+          podFQDNs: Required
+    - name: ZOOKEEPER_METRICS_MONITOR
+      value: {{ .Values.metrics.enabled | quote }}
+    - name: ZOOKEEPER_METRICS_PORT
+      value: {{ .Values.metrics.port | quote }}
+    - name: ZOOKEEPER_DATA_DIR
+      value: {{ .Values.zookeeper.dataDir }}
+    - name: ZOOKEEPER_DATA_LOG_DIR
+      value: {{ .Values.zookeeper.dataLogDir }}
+    - name: ZOOBINDIR
+      value: "/opt/bitnami/zookeeper/bin"
+    - name: ZOOBIN
```

Comment:
Is this var useful? Its value is the same as the upper one.

Reply:
> Is this var useful? Its value is the same as the upper one.

It should be necessary. This addon uses the Bitnami image, and the Bitnami image internally has some dependent environment variables (env).

Reply:
> > Is this var useful? Its value is the same as the upper one.
> 
> It should be necessary. This addon uses the Bitnami image, and the Bitnami image internally has some dependent environment variables (env).

@kubeJocker please take a look

Reply:
This var is used in the ZkServer.sh of version 3.4, so it is necessary.

Reply:
Got it, please add a comment as well.

Reply:
> Got it, please add a comment as well.

done